Example/EmbeddedCropViewController.swift

```diff
@@ -76,8 +75,8 @@ extension EmbeddedCropViewController: CropViewControllerDelegate {
     }
     
     func cropViewControllerDidEndResize(_ cropViewController: CropViewController, original: UIImage, cropInfo: CropInfo) {
-        let croppedImage = Mantis.crop(image: original, by: cropInfo)
-        self.resolutionLabel.text = getResolution(image: croppedImage)
```

Comment:
Why the `Int(size.width) + 1`?

Reply:
I guess it's ceiling the value, right? If so should not we do this in the library itself? So that as a consumer you can just use `size.width`

Reply:
<img width="750" alt="image" src="https://user-images.githubusercontent.com/26723384/179765026-30fe947e-056e-4c49-8f9c-969d0fcd18e6.png">
I am using floor() to get output image size, it might be one pixel different from what we expect. I think if user want to limit crop size, for the upper limit, user should use `Int(size.width) + 1` and `Int(size.height) + 1` to make sure the output size is in the valid range.

I will update code to make it less confusing.